scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -182,75 +195,79 @@ function doImportProject() {
   local setup_ws_dir="${WORKSPACE_PATH}/.metadata/.plugins/org.eclipse.oomph.setup"
   local setup_ws_name="workspace.setup"
   local git_url=""
-  local proj_dir="${2:-${CWD}}" # if no 2nd argument: defaults to current working directory
-  local proj_name="$(basename ${2:-${CWD}})" # if no 2nd argument: defaults to current working directory
+  local proj_dir="${2:-${CWD}}" # if no 2nd argument: default to current working directory
+  local proj_name=$(basename "${proj_dir}") # derive project name from project directory
 
   if [ "${1}" = "mvn" ]
   then
-    if doIsGitHttpsUrl "${2}" # if ${2} is a Git-URL, derive several values from it, select correct setup template
+    if doIsGitHttpsUrl "${2}" # if ${2} is Git-URL, derive several values, select correct setup template
     then
       git_url="${2}"
-      proj_name="$(basename ${git_url} .git)"
+      proj_name=$(basename "${git_url}" .git)
       proj_dir="${WORKSPACE_PATH}/${proj_name}"
       setup_templ_name="mvn_clone.setup"
     else
       setup_templ_name="mvn.setup"
     fi
   elif [ "${1}" = "plain" ]
   then
-    if doIsGitHttpsUrl "${2}" # if ${2} is a Git-URL, derive several values from it, select correct setup template
+    if doIsGitHttpsUrl "${2}" # if ${2} is Git-URL, derive several values, select correct setup template
     then
       git_url="${2}"
-      proj_name="$(basename ${git_url} .git)"
+      proj_name=$(basename "${git_url}" .git)
       proj_dir="${WORKSPACE_PATH}/${proj_name}"
       setup_templ_name="plain_clone.setup"
     else
       setup_templ_name="plain.setup"
     fi
   else
-    echo "You must specify the project type to import (mvn/plain)!"
+    echo "You must specify the type of project to import (mvn/plain)!"
   fi
-
+  
   local setup_templ="${setup_templ_dir}/${setup_templ_name}"
   local setup_ws="${setup_ws_dir}/${setup_ws_name}"
-  
-  echo "SETUP_TEMPL: ${setup_templ}" #DEBUG
-  echo "PROJ_NAME:   ${proj_name}" #DEBUG
-  echo "PROJ_DIR:    ${proj_dir}" #DEBUG
+
+  if command -v cygpath >/dev/null # if cygwin/git bash are used (Windows only)
+  then
+    proj_dir=$(cygpath -m "${proj_dir}") # convert Unix-style path to Windows-style path
+    proj_dir=$(echo "${proj_dir}" | cut -d':' -f2-) # if existent, cut drive letter
+  fi
```

Comment:
I'm using `cygpath` now to convert the path format and cut the drive letter based on the existence of a `:` in the path (which is only ever the case under Windows).

One (minor) problematic result of this approach is, that if the user enters something like `/myDir/subDir` under Windows, `cygpath` will turn it into `C:/Program Files/Git/myDir/subDir` (because no drive was defined, `cygpath` defaults to the Git/Cygwin directory).

If the user correctly enters `/c/myDir/subDir`, everything works as expected:
* `cygpath` turns the path into `C:/myDir/subDir` (mixed format)
* using `cut` and the delimiter `:` the leading drive letter is removed

I doubt that many people will make use of the option to enter an absolute path, since `cd`'ing to the desired directory is far more convenient (which doesn't excuse the problematic behaviour of course). However, as stated in the initial [issue description](https://github.com/devonfw/ide/issues/175#issue-479992948), this functionality could be useful in certain cases, so I wanted to support it.